getting_started.Rmd

```diff
@@ -74,80 +83,89 @@ In the pop up window, search for and add `jhudsl-robot`.
+By default, Github Pages is sent to a url that looks like `username.github.io` where `username` is either your individual user name or your organization's name. However, you can customize your GitHub Pages URL (like we have in this example here - `jhudatascience.org`). So it is likely your pages URL is something like `username.github.io/your_ottr_repository_name`.
 
 For more information about GitHub pages we recommend you reference the [GitHub documentation here](https://docs.github.com/en/pages/quickstart).
 
-\* **Warning**: if you go to your pages url right now (before you've pushed any file changes) you might see a 404 error because nothing has been triggered to be rendered. You can check your url pages after you file your first pull request (which we will guide you through [in the next section](https://github.com/jhudsl/OTTR_Template/wiki/Start-editing-your-course)).
+**Warning**: if you go to your pages url right now (before you've pushed any file changes) you might see a 404 error because nothing has been triggered to be rendered. You can check your url pages after you file your first pull request (which we will guide you through [in the next section](https://github.com/jhudsl/OTTR_Template/wiki/Start-editing-your-course)).
+
 
 ### 9. Set up branch rules
 
 \* Don't set these settings right away! Wait a few minutes after starting your repository.
 
 Go to `Settings` > `Branches` and click `Add rule`. This is only possible if you have GitHub Pro, Team, or Enterprise access. We recommend seeing if your organization can give you access if you do not already have access. To get this access through your organization, you will need to transfer ownership of your repo to your organization. See [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository) about how to do this.
 
-![Add branch rule](https://raw.githubusercontent.com/jhudsl/OTTR_Template/main/resources/screenshots/branches.png)
+![](https://raw.githubusercontent.com/jhudsl/OTTR_Template/main/resources/screenshots/branches.png)
 
 For `Branch name pattern`, put `main`.
 
-![Put main for branch name pattern](https://raw.githubusercontent.com/jhudsl/OTTR_Template/main/resources/screenshots/main.png)
+![](https://raw.githubusercontent.com/jhudsl/OTTR_Template/main/resources/screenshots/main.png)
 
 _Protect the main branch_:  
-Then check the box that says `Require pull requests before merging`.
+Then check the box that says `Require a pull request before merging`.
 
-\* Note that if the user whose personal access token does not have admin privileges for step `6. Set up your GitHub personal access token`, then you will need to skip this setting.
+\* Note that if the user whose personal access token does not have admin privileges for step `6. Set up your GitHub personal access token`, then you will need to skip this setting. (TODO: Clarify)
```

Comment:
What needs to be clarified here?

Reply:
I transferred my repo to jhudsl organization, https://github.com/jhudsl/OTTR-Course, and now it looks like I can't change the settings.

Reply:
You should have access now.